docs/Modders/Creating-a-custom-tileset.md

```diff
@@ -57,9 +74,27 @@ A color defined with normalized RGBA values. Default value: "{"r":0, "g":0, "b":
 
 Defines the color of the fog of war. The color gets approximated by 60% to allow the colors of the images below to shine through.
 
+### fallbackTileSet
+
+A string value. Default value: "FantasyHex"
```

Comment:
This default value was also changed to HexaRealm, right?

Reply:
I haven't changed the default value for fallbackTileset (TilesetConfig.kt L11) because "Hexarealm" does not contain Unit sprites yet and uses "FantasyHex" as fallbackTileset itself.